packages/coin-tester/src/components/coins/avaxc/index.tsx

```diff
@@ -0,0 +1,447 @@
+import React, { useState, useEffect } from 'react';
+import { Container } from 'react-bootstrap';
+import { Transport, apdu, utils, config } from '@coolwallet/core';
+import { NoInput, TwoInputs } from '../../../utils/componentMaker';
+
+const Web3 = require('web3');
+const sigUtil = require('eth-sig-util');
+
+const web3 = new Web3('https://api.avax-test.network/ext/bc/C/rpc');
+
+import Template from '@coolwallet/avaxc';
+
+interface Props {
+	transport: Transport | null;
+	appPrivateKey: string;
+	appPublicKey: string;
+	isLocked: boolean;
+	setIsLocked: (isLocked: boolean) => void;
+}
+
+function CoinTemplate(props: Props) {
+	const temp = new Template();
+	const [address, setAddress] = useState('');
+	const [signedTransaction, setSignedTransaction] = useState('');
+	const [signedERC20Transaction, setSignedERC20Transaction] = useState('');
+	const [signedSmartContractTransaction, setSignedSmartContractTransaction] = useState('');
+	const [signedMessage, setSignedMessage] = useState('');
+	const [signedTypedData, setSignedTypedData] = useState('');
+	const [value, setValue] = useState('0');
+	const [to, setTo] = useState('');
+	const [data, setData] = useState('');
+
+	const { transport, appPrivateKey } = props;
+	const disabled = !transport || props.isLocked;
+
+	const handleState = async (request: () => Promise<string>, handleResponse: (response: string) => void) => {
+		props.setIsLocked(true);
+		try {
+			const response = await request();
+			handleResponse(response);
+		} catch (error) {
+			handleResponse(error.message);
+			console.error(error);
+		} finally {
+			props.setIsLocked(false);
+		}
+	};
+
+	const getAddress = async () => {
+		handleState(async () => {
+			const appId = localStorage.getItem('appId');
+			if (!appId) throw new Error('No Appid stored, please register!');
+			const address = await temp.getAddress(transport!, appPrivateKey, appId, 0);
+			return address;
+		}, setAddress);
+	};
+
+	const signTransaction = async () => {
+		handleState(async () => {
+			const transaction = {
+				nonce: web3.utils.toHex(await web3.eth.getTransactionCount(address, 'pending')),
+				gasPrice: web3.utils.toHex(await web3.eth.getGasPrice()),
+				gasLimit: web3.utils.toHex(8000000),
+				to: to,
+				value: web3.utils.toHex(web3.utils.toWei(value.toString(), 'ether')),
+				data: '',
+			};
+
+			const appId = localStorage.getItem('appId');
+			if (!appId) throw new Error('No Appid stored, please register!');
+			const signedTx = await temp.signTransaction(transport!, appPrivateKey, appId, 0, transaction);
+			await web3.eth.sendSignedTransaction(signedTx);
+			return signedTx;
+		}, setSignedTransaction);
+	};
+
+	const signERC20Transaction = async () => {
```

Comment:
You can generate data without `abi`. Please refer to the other EVM compatible chain in `coin-tester`.